From 03f6263010ab60a37f1bfd2752b4066ba173fff3 Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Thu, 30 Apr 2020 06:54:36 +0300
Subject: [PATCH 132/145] improving statusbar icon spacing overlays

	modified:   core/res/res/values/dimens.xml
	modified:   packages/SystemUI/res/values/dimens.xml
	modified:   packages/overlays/StatusBar2Overlay/res/values/dimens.xml
	modified:   packages/overlays/StatusBar3Overlay/res/values/dimens.xml

Change-Id: Ie0b2dba8110a00f28904bf8af3339f91b51bc0db
---
 core/res/res/values/dimens.xml                            | 2 ++
 packages/SystemUI/res/values/dimens.xml                   | 8 ++++----
 packages/overlays/StatusBar2Overlay/res/values/dimens.xml | 2 ++
 packages/overlays/StatusBar3Overlay/res/values/dimens.xml | 2 ++
 4 files changed, 10 insertions(+), 4 deletions(-)

diff --git a/core/res/res/values/dimens.xml b/core/res/res/values/dimens.xml
index b5281c9c713..12e3ba6df13 100644
--- a/core/res/res/values/dimens.xml
+++ b/core/res/res/values/dimens.xml
@@ -82,6 +82,8 @@
     <dimen name="status_bar_edge_ignore">5dp</dimen>
     <!-- Default horizontal drawable padding for status bar icons -->
     <dimen name="status_bar_horizontal_padding_core">0dp</dimen>
+    <!-- Default extra horizontal drawable padding for status bar icons -->
+    <dimen name="status_bar_horizontal_padding_extra">2dp</dimen>
 
     <!-- Default radius of the software rounded corners. -->
     <dimen name="rounded_corner_radius">0dp</dimen>
diff --git a/packages/SystemUI/res/values/dimens.xml b/packages/SystemUI/res/values/dimens.xml
index 9e6ccabe2db..faf3304e096 100644
--- a/packages/SystemUI/res/values/dimens.xml
+++ b/packages/SystemUI/res/values/dimens.xml
@@ -76,7 +76,7 @@
     <dimen name="status_bar_clock_size">14sp</dimen>
 
     <!-- The starting padding for the clock in the status bar. -->
-    <dimen name="status_bar_clock_starting_padding">1dp</dimen>
+    <dimen name="status_bar_clock_starting_padding">@*android:dimen/status_bar_horizontal_padding_extra</dimen>
 
     <!-- The end padding for the clock in the status bar. -->
     <dimen name="status_bar_clock_end_padding">0dp</dimen>
@@ -85,7 +85,7 @@
     <dimen name="status_bar_left_clock_starting_padding">0dp</dimen>
 
     <!-- End padding for left-aligned status bar clock -->
-    <dimen name="status_bar_left_clock_end_padding">1dp</dimen>
+    <dimen name="status_bar_left_clock_end_padding">@*android:dimen/status_bar_horizontal_padding_extra</dimen>
 
     <!-- Spacing after the wifi signals that is present if there are any icons following it. -->
     <dimen name="status_bar_wifi_signal_spacer_width">@dimen/status_bar_horizontal_padding</dimen>
@@ -823,10 +823,10 @@
     <dimen name="signal_cluster_margin_start">@dimen/status_bar_horizontal_padding</dimen>
 
     <!-- Padding between signal cluster and battery icon -->
-    <dimen name="signal_cluster_battery_padding">2dp</dimen>
+    <dimen name="signal_cluster_battery_padding">@*android:dimen/status_bar_horizontal_padding_extra</dimen>
 
     <!-- Padding for signal cluster and battery icon when there are not icons in signal cluster -->
-    <dimen name="no_signal_cluster_battery_padding">2dp</dimen>
+    <dimen name="no_signal_cluster_battery_padding">@*android:dimen/status_bar_horizontal_padding_extra</dimen>
 
     <!-- Screen pinning request width -->
     <dimen name="screen_pinning_request_width">@dimen/match_parent</dimen>
diff --git a/packages/overlays/StatusBar2Overlay/res/values/dimens.xml b/packages/overlays/StatusBar2Overlay/res/values/dimens.xml
index 0c6284d42ad..1d42bc2028a 100644
--- a/packages/overlays/StatusBar2Overlay/res/values/dimens.xml
+++ b/packages/overlays/StatusBar2Overlay/res/values/dimens.xml
@@ -19,6 +19,8 @@
 <resources>
     <!-- Default horizontal drawable padding for status bar icons -->
     <dimen name="status_bar_horizontal_padding_core">2dp</dimen>
+    <!-- Default extra horizontal drawable padding for status bar icons -->
+    <dimen name="status_bar_horizontal_padding_extra">4dp</dimen>
     <!-- Height of notification icons in the status bar -->
     <dimen name="status_bar_icon_size">21dip</dimen>
     <!-- Intrinsic size of most system icons in status bar. This is the default value that
diff --git a/packages/overlays/StatusBar3Overlay/res/values/dimens.xml b/packages/overlays/StatusBar3Overlay/res/values/dimens.xml
index 2bdbf325b85..4899aa72949 100644
--- a/packages/overlays/StatusBar3Overlay/res/values/dimens.xml
+++ b/packages/overlays/StatusBar3Overlay/res/values/dimens.xml
@@ -19,6 +19,8 @@
 <resources>
     <!-- Default horizontal drawable padding for status bar icons -->
     <dimen name="status_bar_horizontal_padding_core">3dp</dimen>
+    <!-- Default extra horizontal drawable padding for status bar icons -->
+    <dimen name="status_bar_horizontal_padding_extra">6dp</dimen>
     <!-- Height of notification icons in the status bar -->
     <dimen name="status_bar_icon_size">23dip</dimen>
     <!-- Intrinsic size of most system icons in status bar. This is the default value that
-- 
2.17.1

